operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -57,24 +56,24 @@ The goal is not to have the same release cadence for all products, but there may
    * Enables us to avoid missing key tasks and to avoid last minute work.
    * E.g. When opening a PR, add a “Docs/Needed” label for any PRs that need docs and communicate to Tech Writers. Don’t wait until merging the PR/feature. This ensures that we have time to complete docs on time when our releases become faster.
    * Make tracking bugs and testing requirements easy. E.g:
-   * Resolve Jira tickets for QA when PRs are merged \(and cherry-picked\).
-   * Add QA test steps to Jira tickets and/or PRs.
-   * Add Fix Versions and Milestones in Jira/PRs for bugs/tickets for easy tracking.
-   * Add clear Release Notes on PRs.
+     * Resolve Jira tickets for QA when PRs are merged \(and cherry-picked\).
```

Comment:
Question: what do the ``\`` symbols stand for, indicate, or affect in terms of display?